models/rnn.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import theano.tensor as T
+from theano import scan
+from theano.ifelse import ifelse
+
+from smartpy import Model
+from smartpy.misc.utils import sharedX
+from .utils import WeightsInitializer
+
+
+class RNN(Model):
+    def __init__(self, input_size, output_size, hidden_layers, output_act_fct=lambda x: x, dropout_rate=0.0,
+                 use_batch_normalization=False, seed=1234):
+        self.tWs = []
+        self.tbs = []
+        self.tVs = [] # recurrent connections
+        self.th0 = []
+        self.act_fcts = []
+        self.act_fcts_param = []
+        self.batch_normalization_param = []
+
+        self.dropout_rate = sharedX(dropout_rate, name='dropout_rate')
+        self.use_dropout = sharedX(1.0 if dropout_rate > 0 else 0.0, name='use_dropout?')
+        self.use_batch_normalization = use_batch_normalization
+        self._trng = T.shared_randomstreams.RandomStream(seed)
```

Comment:
Using the previously imported `RandomStreams` this line should be `self._trng = RandomStreams(seed)`